examples/ecs/one_shot_systems.rs

```diff
@@ -0,0 +1,73 @@
+//! Demonstrates the use of "one-shot systems", which run once when triggered.
+//!
+//! These can be useful to help structure your logic in a push-based fashion,
+//! reducing the overhead of running extremely rarely run systems
+//! and improving schedule flexibility.
+//!
+//! See the [`SystemRegistry`](bevy::ecs::SystemRegistry) docs for more details.
+
+use bevy::{
+    ecs::system::{SystemId, SystemRegistry},
+    prelude::*,
+};
+
+fn main() {
+    App::new()
+        .add_startup_system(count_entities)
+        .add_startup_system(setup)
+        // One shot systems are interchangeable with ordinarily scheduled systems.
+        // Change detection, Local and NonSend all work as expected.
+        .add_system(count_entities.in_base_set(CoreSet::PostUpdate))
+        // One-shot systems can be used to build complex abstractions
+        // to match the needs of your design.
+        // Here, we model a very simple component-linked callback architecture.
+        .add_system(evaluate_callbacks)
+        .run();
+}
+
+// Any ordinary system can be run via commands.run_system or world.run_system.
+//
+// Chained systems, exclusive systems and systems which themselves run systems cannot be called in this way.
+fn count_entities(all_entities: Query<()>) {
+    dbg!(all_entities.iter().count());
+}
+
+#[derive(Component)]
+struct Callback(SystemId);
+
+#[derive(Component)]
+struct Triggered;
+
+fn setup(mut system_registry: ResMut<SystemRegistry>, mut commands: Commands) {
+    commands.spawn((
+        // The Callback component is defined in bevy_ecs,
```

Comment:
I don't think it is? Does this still need to be done or should this comment be changed?